ntc_templates/templates/huawei_vrp_display_vlan.textfsm

```diff
@@ -40,3 +48,12 @@ STATUS
   ^${VLAN_ID}\s+${VLAN_STATUS}\s+\S+\s+\S+\s+\S+\s+${VLAN_NAME} -> Continue.Record
   ^\d+ -> Clearall
   ^. -> Error
+
+
+VLANS_AP
```

Comment:
The convention I've seen across templates has been single blank lines between sections.
So here I'm suggesting the minor edit to remove one of those two before the `VLANS_AP` State

By comparison there was already a `STATUS` State section preceded with two blank lines, which could also be reduced to a single blank line if don't mind. :grin: Thank you!

```suggestion

VLANS_AP
```